script/efr32-definitions

```diff
@@ -35,9 +35,16 @@ if [[ -n ${BASH_SOURCE[0]} ]]; then
 else
     script_path="$0"
 fi
-script_dir="$(realpath "$(dirname "${script_path}")")"
-repo_dir="$(dirname "${script_dir}")"
-sdk_dir="${repo_dir}/third_party/silabs/gecko_sdk"
+
+# Check if repo_dir is defined
+if [ -z ${repo_dir+x} ]; then
```

Comment:
What is the `+x` modifier used in `${repo_dir+x}` and `${gsdk_dir+x}`?

Reply:
https://stackoverflow.com/a/13864829